specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/privateEndpoint.json

```diff
@@ -116,22 +116,22 @@
         ],
         "responses": {
           "200": {
-            "description": "Request successful. The operation returns the resulting InterfaceEndpoint resource.",
+            "description": "Request successful. The operation returns the resulting PrivateEndpoint resource.",
             "schema": {
-              "$ref": "#/definitions/InterfaceEndpoint"
+              "$ref": "#/definitions/PrivateEndpoint"

```

Comment:
Please add the "default" response to model errors, i.e. :
``
          "default": {
            "description": "Error",
            "schema": {
              "$ref": "./network.json#/definitions/Error"
            }
          }
``